android/brave_java_resources.gni

```diff
@@ -175,6 +175,7 @@ brave_java_resources = [
   "java/res/drawable-nodpi/ic_quick_action_search_and_bookmark_widget_preview.png",
   "java/res/drawable-nodpi/loader0.png",
   "java/res/drawable-nodpi/loader0_orange.png",
+  "java/res/drawable-nodpi/ramp.png",
```

Comment:
could we have it in svg format? We are going to convert all pictures to vector graphic soon. That will decrease resources size plus they are scalable. It doesn't make sense to add anything new via png.

Reply:
Extracted from iOS, so the only option available was PNG. I'm going to try to manually convert this.

Reply:
Or ask design team for a help

Reply:
The results were low quality, I'll ask the design team for some help.